WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/UsernameChangerFullScreenDialogFragment.java

```diff
@@ -187,9 +187,13 @@ public void onStop() {
     public boolean onConfirmClicked(FullScreenDialogController controller) {
         ActivityUtils.hideKeyboard(getActivity());
 
-        Bundle result = new Bundle();
-        result.putString(RESULT_USERNAME, mUsernamesAdapter.mItems.get(mUsernamesAdapter.getSelectedItem()));
-        controller.confirm(result);
+        if (mUsernamesAdapter != null && mUsernamesAdapter.mItems != null) {
+            Bundle result = new Bundle();
+            result.putString(RESULT_USERNAME, mUsernamesAdapter.mItems.get(mUsernamesAdapter.getSelectedItem()));
+            controller.confirm(result);
+        } else {
+            controller.dismiss();
```

Comment:
would it make sense to log something here?

Reply:
I don't think so.  What happens in this case is the user closes the full-screen dialog before the list finishes loading.  I don't think there is anything significant we can log there.  Do you?